src/containers/Review/ReviewPage.tsx

```diff
@@ -131,14 +131,11 @@ const ReviewPage: React.FC<{
   const isSubmitted =
     thisReview?.current.reviewStatus === ReviewStatus.Submitted ||
     thisReview?.current.reviewStatus === ReviewStatus.ChangesRequested
-  const isLocked =
-    thisReview?.current.reviewStatus == ReviewStatus.Locked ||
-    thisReview?.current.reviewStatus === ReviewStatus.Discontinued
+  const isLocked = thisReview?.isLocked
```

Comment:
Hey, just wondering if we should keep this check for Discontinued too?

Reply:
I'm not actually sure what "Discontinued" means.

When we review the full branch PR, there's a few places where we've pulled out stuff to do with "ASSIGN.LOCKED" or "is_locked" that we'll need to double-check (like this). Need to make sure that the checks being done are handled in the new case.